proto/ibc/core/channel/v1/tx.proto

```diff
@@ -266,9 +266,10 @@ message MsgChannelUpgradeInit {
 message MsgChannelUpgradeInitResponse {
   option (gogoproto.goproto_getters) = false;
 
-  string  channel_id       = 1;
-  Upgrade upgrade          = 2 [(gogoproto.nullable) = false];
-  uint64  upgrade_sequence = 3;
+  string             channel_id       = 1;
+  Upgrade            upgrade          = 2 [(gogoproto.nullable) = false];
+  uint64             upgrade_sequence = 3;
+  ResponseResultType result           = 4;
```

Comment:
Wondering if we should make `result` the first field, just in case in the future we want to add more fields to the response and `result` is not in between.

Reply:
yeah, I thought about this myself. I think we should definitely raise the point during final walk-through/audit. willing to make the change here if anyone else shares the opinion.